ysh/grammar.pgen2

```diff
@@ -597,19 +597,15 @@ regex: [re_alt] (('|'|'or') re_alt)*
 case_pat: (
     pat_paren
```

Comment:
I think it would be more readable to inline pat_paren, like

```
ysh_case_pat =
  '(' ( 'else' ')' | pat_exprs )
| pat_eggex
```

So we see the 3 alternatives in one place

Reply:
Sure. I decided to leave the separate `pat_else` because I don't think it reduces readability while making the translation more explicit. Now we have:

```
ysh_case_pat: (
    '(' (pat_else | pat_exprs)
  | pat_eggex
)
```

If you still think it would be better to embed the `pat_else`, then we can change that.